src/libs/actions/Report.ts

```diff
@@ -654,9 +653,6 @@ function updateGroupChatName(reportID: string, reportName: string) {
             key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
             value: {
                 reportName: ReportConnection.getAllReports()?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`]?.reportName ?? null,
-                errors: {
```

Comment:
I am not sure how our problem gets solved by not setting `errors` here. The problem mentioned [here](https://github.com/Expensify/App/pull/44130#discussion_r1675708180) still exists. When `openReport` is called, `errorFields` comes back as follows with null (as shown below) and, strangely, FE resets the `errorFields` in report in Onyx. Don’t we need to get to the bottom of this?

![Screenshot 2024-07-20 at 11 17 08 PM](https://github.com/user-attachments/assets/f850f437-c1bf-4cb8-b9da-a597244417e6)